packages/gatsby/src/commands/develop.ts

```diff
@@ -61,7 +84,7 @@ onExit(() => {
 })
 
 const waitJobsFinished = () =>
-  new Promise((resolve, reject) => {
+  new Promise(resolve => {
```

Comment:
I'm a little surprised that TS doesn't care that we don't listen to rejections here. I think node is going to crash hard for that in a future version, whenever that is.

Reply:
Isn't that about the listeners more than the invocation? I thought node was going to start throwing if you did `promise.then()` and not `promise.then().catch()`

Reply:
Oh, TIL :)